config/install/field.field.node.vehicles.field_consum.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_consum
+    - node.type.vehicles
+id: node.vehicles.field_consum
+field_name: field_consum
+entity_type: node
+bundle: vehicles
+label: Consumables
+description: 'The maximum length of time that this vehicle can provide consumables for its entire crew without having to resupply'
```

Comment:
Add a period to the end of the help text for consistency.
![vehicles_field_consumables_help_text](https://github.com/notS0Solo/swapimigration/assets/1434979/87b3f005-4c07-4cff-a5b2-e002f29889e7)

Reply:
Fixed